### ISSUE TITLE
Problems when try to run inference

### ISSUE THREAD

**Issue by superkido511** [10Ring/FakeSTormer#4]
I'm trying to run the test.py for a single image using the c0 Fakestromer checkpoint (TopDownDetector_C0_ViTBase224_ST_hm100_tempLOC0.2_4SBI_SAM_mp0.01_temp2_0vidAug_0.25_temp_normlms_IN_model_best.pth) 
that I got from here: https://www.dropbox.com/scl/fo/elk2szqf0du4l6zm5job9/AAdVmNH--6ywHBZGNQJlR5o?rlkey=j8xesf2fu4ahxdw99w5ndrkb2&e=1&st=fe6drzpx&dl=0
I use the config file `configs/temporal/FakeSFormer_base_c0.yaml` and task `test_img`.
When I try to run test.py, I got this error Not enough value to unpack at:
https://github.com/10Ring/FakeSTormer/blob/main/models/networks/backbones/vit.py#L644

I tried added the time dimension by adding `img_trans = torch.unsqueeze(img_trans, 2)` below this line:
https://github.com/10Ring/FakeSTormer/blob/main/scripts/test.py#L127
but got the shape mismatch error.

Has anyone actually success in running inference?

**Comment by li129031** [comment]
@superkido511 Is that so? I actually didn't use the c0 Fakestromer checkpoint for testing, however, I successfully ran the inference script with the c23 Fakestromer checkpoint.

**Comment by superkido511** [comment]
> [@superkido511](https://github.com/superkido511) Is that so? I actually didn't use the c0 Fakestromer checkpoint for testing, however, I successfully ran the inference script with the c23 Fakestromer checkpoint.

@li129031 Sorry to bother you, I tried the C23 checkpoint but got the same error. Could you clarify which argument and config file did you use?

**Comment by li129031** [comment]
@superkido511 I completed the tests following the configuration file mentioned by the author in the README. Also, do you happen to know how the DFDC and DFD datasets are processed?

**Comment by superkido511** [comment]
@li129031 I assume it's the dataset are processed same way as mentioned in the original competition unless stated otherwise
https://www.kaggle.com/competitions/deepfake-detection-challenge/code

**Comment by li129031** [comment]
@superkido511 So far, I've only managed to find the 470GB DFDC training set from the competition, and it seems there’s no public test set. I'm still confused about how people process the data to get a test set, since I didn't see any details regarding this on the competition website.

**Comment by superkido511** [comment]
> [@superkido511](https://github.com/superkido511) So far, I've only managed to find the 470GB DFDC training set from the competition, and it seems there’s no public test set. I'm still confused about how people process the data to get a test set, since I didn't see any details regarding this on the competition website.

Did you download it from here?
https://ai.meta.com/datasets/dfdc/

**Comment by li129031** [comment]
@superkido511 Yes, that's the link I used.

**Comment by superkido511** [comment]
> [@superkido511](https://github.com/superkido511) Yes, that's the link I used.

They mentioned in the paper that the public and private test set are not released. This repo use cross validation on the entire dataset then take the average. As for the detail on how the folds are split, I think it's in test.py with "eval" mode.

**Comment by li129031** [comment]
@superkido511 ,have you attempted to train this model yourself? If so, did the training process run smoothly for you?"
